tests/rustdoc/doc_auto_cfg_reexports.rs

```diff
@@ -0,0 +1,22 @@
+// Checks that `cfg` are correctly applied on inlined reexports.
```

Comment:
Shouldn't this also check that `cfg`s *don't* get applied on non-inlined reexports?